huaweicloud/services/modelarts/resource_huaweicloud_modelarts_model.go

```diff
@@ -0,0 +1,726 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product ModelArts
+// ---------------------------------------------------------------
+
+package modelarts
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceModelartsModel() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceModelartsModelCreate,
+		ReadContext:   resourceModelartsModelRead,
+		DeleteContext: resourceModelartsModelDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(20 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"region": {
```

Comment:
please adjust the order of the arguments, like the doc, to make this code easier to read

Reply:
ok